CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 
 * Changed `ReferralRequest` mapping to use `ASAP` instead of the `Stat` value.
+* Changed 'author' to be used as Recorder and 'practitioner' to be used as Asserter in AllergyIntolerance
```

Comment:
This should be under the `Fixed` heading, [see my rant here](https://github.com/NHSDigital/nia-patient-switching-standard-adaptor/pull/374#discussion_r1399363510). 🤣

Reply:
makes sense, will try to remember to move it under a more correct category with my next PR.